pkg/datastore/mongodb/mongodb.go

```diff
@@ -59,7 +59,7 @@ func (m *mongodb) Put(ctx context.Context, kind, name string, entity interface{}
 func (m *mongodb) Find(ctx context.Context, kind string) (datastore.Iterator, error) {
 	collection := m.client.Database(m.database).Collection(kind)
 	// bson.D{{}} specifies 'all documents'
```

Comment:
I don't understand why here  a double  “{{}}”  can specifies all documents
the usage in  https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo may be a single "{}"